apps/meteor/app/lib/server/functions/saveUser.ts

```diff
@@ -409,10 +407,9 @@ export const saveUser = async function (userId, userData) {
 		sendPassword = false;
 	}
 
-	const updateUser = {
-		$set: {},
-		$unset: {},
-	};
+	const updateUser: DeepWritable<UpdateFilter<IUser>> = {};
```

Comment:
this seems a good opportunity for using the new `Updater` https://adr.rocket.chat/0099

Reply:
Done!